README.md

```diff
@@ -58,8 +59,8 @@ to supply valid UUID's when testing.
 For testing, you can upload from the **scripts** folder, you should be able to run;
 
 ```bash
-ce=`python3 -c "from uuid import uuid4; print(uuid4())"`
-./upload.py local $ce 30
+ce=`pipenv run python -c "from uuid import uuid4; print(uuid4())"`
+pipenv run ./upload.py local $ce 30
```

Comment:
Looks like the port in this script is wrong. It points to 8080 but the app runs up on 8082 and then I get this error

```
%% upload error "404" - "{
  "detail": "The requested URL was not found on the server.  If you entered the URL manually please check your spelling and try again.",
  "status": 404,
  "title": "Not Found",
  "type": "about:blank"
}
"
```

Reply:
Probably not worth looking into it on this PR